nlp/admin.js

```diff
@@ -44,12 +47,56 @@ router.post('/reload/exact/@:model_tag/:locale', (req, res, next) => {
     }).catch(next);
 });
 
-router.post('/reload/@:model_tag/:locale', (req, res, next) => {
+
+function execCommand(script, argv) {
+    return new Promise((resolve, reject) => {
+        const stdio = ['ignore', 'pipe', 'pipe'];
```

Comment:
Because we're not parsing the output of the command (like we do in training), this can be simplified to be `['ignore', 'inherit', 'inherit']`, which lets us remove the `child.stdio[]` code later.